tests/Database/RelationsTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+class RelationsTest extends DbTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->create_tables();
+        $this->seed_tables();
```

Comment:
@mjauvin Convention is to use camelcase for method names
```suggestion
        $this->createTables();
        $this->seedTables();
```

Reply:
yeah, don't know where that came from... 8)